dora/core/common/src/main/java/alluxio/underfs/AtomicFileOutputStream.java

```diff
@@ -81,9 +81,9 @@ public void close() throws IOException {
         throw new IOException(
             ExceptionMessage.FAILED_UFS_RENAME.getMessage(mTemporaryPath, mPermanentPath));
       }
-    } finally {
+    } catch (IOException e) {
       if (!mUfs.deleteFile(mTemporaryPath)) {
-        LOG.error("Failed to delete temporary file {}", mTemporaryPath);
+        LOG.warn("Failed to delete temporary file {}", mTemporaryPath);
```

Comment:
so this masks the original ioexception?

Reply:
yeah, it's best effort delete tmp files after fail to rename

Reply:
no, meaning that this is not throwing the original IOException out?

Reply:
oh I see, thanks for finding this mistake!